gammapy/data/metadata.py

```diff
@@ -32,6 +32,13 @@
 METADATA_FITS_KEYS["observation"] = OBSERVATION_METADATA_FITS_KEYS
 
 
+EVENTLIST_METADATA_FITS_KEYS = {
+    "event_class": "EV_CLASS",
```

Comment:
what is the evt class?
In parallel, do we have event types? If yes, the evt types should be a list, in general. Is not it?

Reply:
It depends if one event list is provided per event type or if event list is a general object with a single column.
The SDC will use the former, but in general we might be better off with the latter.

In all generality, it could be a list of `str` indeed.

Reply:
So you'd prefer the `event_class` to be a list of `str` instead of a `str`?

From the formatting we could actually have two fields here `EV_CLASS` (same for all events in a given event list) and `EVENT_TYPE` (can have a different value for each event).
https://gamma-astro-data-formats.readthedocs.io/en/v0.3/events/events.html#iact-events-class-type

Reply:
Maybe we can start with the most minimal requirement for the SDC, and then allow lists in the future? Otherwise, if we decide otherwise, we would need to through the entire deprecation process?